docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -477,9 +474,48 @@ Where,
 
 	- rr_i is the relative risk assigned to the individual simulant
 
-.. todo::
+The pseudo-code used to implement the vitamin A intervention effect in Vivarium is shown below:
+
+
+.. code-block:: Python
+
+	# Definitions
+	effectively_covered := individual is recieving fortification and it is affecting their 
+					probability of being vitamin a deficient.
+	t                   := current time
+
+	# Population level params
+	rr                = 1 if under 6 months, else 2.22 (or whatever)
+	coverage(t)       = baseline coverage of vitamin a fortification
+	exposure(t)       = baseline exposure to lack of vitamin a fortification 
+	                  = 1 - coverage(t)
+	delta_coverage(t) = intervention shift in baseline coverage of vitamin a fortification
+	coverage*(t)      = coverage(t) + delta_coverage(t)
+	exposure*(t)      = 1 - coverage*(t)
+	time_lag          = time from when an individual starts eating fortified 
+				food until the effect is present
+	mean_rr(t)        = rr * exposure(t) + 1 * (1 - exposure(t))
+	paf(t)            = (mean_rr(t) - 1)/mean_rr(t)
+	p_gbd(t)          = vitamin a deficiency exposure
+
+	# Individual attributes
+	qx_i           = propensity for an indidual to receive vitamin a fortification
+	x_i(t)         = whether an individual is receiving vitamin a fortification
+	               = qx_i > exposure*(t) 
+	               = qx_i < coverage*(t)   
```

Comment:
Is it possible, there is a tie for exposure and coverage (eg. exposure = coverage = 0.5)?

Reply:
Good point! I will add a note